extensions/classifiers/PythonProgramTokenizer.js

```diff
@@ -265,7 +257,7 @@ oppia.factory('PythonProgramTokenizer', [
               indents = indents.slice(0, -1);
               tokenizedProgram.push([PythonProgramTokenType.DEDENT, '']);
             }
-          }
+          } // eslint-disable-line brace-style
```

Comment:
Ditto, put "// Continued statement." in the line after the "else {"

Reply:
Got it, thanks!